Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+      <alternative track="30"/>
+    </track>
+    <track name="3">
+      <block name="77G3"/>
+      <alternative track="2"/>
+      <alternative track="1"/>
+      <alternative track="4"/>
+      <alternative track="5"/>
+      <alternative track="6"/>
+      <alternative track="30"/>
+    </track>
+    <track name="4">
+      <block name="77G4"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="1"/>
+      <alternative track="5"/>
+      <alternative track="6"/>
+      <alternative track="30"/>
+    </track>
+    <track name="5">
+      <block name="77G5"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+      <alternative track="1"/>
+      <alternative track="6"/>
+      <alternative track="30"/>
+    </track>
+    <track name="6">
+      <block name="77G6"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+      <alternative track="5"/>
+      <alternative track="1"/>
+      <alternative track="30"/>
+    </track>
+    <track name="30">
+      <block name="77G30"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+      <alternative track="4"/>
+      <alternative track="5"/>
+      <alternative track="6"/>
+      <alternative track="1"/>
+    </track>
+    <track name="53">
+      <block name="77G53"/>
+      <alternative track="54"/>
+      <alternative track="55"/>
+      <alternative track="56"/>
+      <alternative track="57"/>
+    </track>
+    <track name="54">
+      <block name="77G54"/>
+      <alternative track="53"/>
+      <alternative track="55"/>
+      <alternative track="56"/>
+      <alternative track="57"/>
+    </track>
+    <track name="55">
+      <block name="77G55"/>
+      <alternative track="54"/>
+      <alternative track="53"/>
+      <alternative track="56"/>
+      <alternative track="57"/>
+    </track>
+    <track name="56">
+      <block name="77G56"/>
+      <alternative track="54"/>
+      <alternative track="55"/>
+      <alternative track="53"/>
+      <alternative track="57"/>
+    </track>
+    <track name="57">
+      <block name="77G57"/>
+      <alternative track="54"/>
+      <alternative track="55"/>
+      <alternative track="56"/>
+      <alternative track="53"/>
+    </track>
+  </station>
+
+  <!--ScheduleFile prüfen, TFH hat 2!-->
+  <station name="Friedrichshafen Hafen" short="TFH" refNr="77" scheduleFile="fn-haf__.abf" localOrderFile="fn-haf__.abf">
+    <track name="152">
+      <block name="7G152"/>
+      <alternative track="153"/>
+    </track>
+    <track name="153">
+      <block name="7G153"/>
+      <alternative track="152"/>
+    </track>
+  </station>
+
+  <station name="Friedrichshafen-Ost" short="TFOS" refNr="77" scheduleFile="fn-ost__.abf">
+    <track name="P2">
+      <block name="77BP2"/>
+    </track>
+  </station>
+
+  <station name="Löwental" short="TLOE" refNr="77" scheduleFile="löwent__.abf">
```

Comment:
Bei diesem und den beiden folgenden Hp's gilt das eben gesagte. Hier ist aufgrund des Blocks keine eindeutige Zuordnung möglich. Man sollte es vermeiden, denselben Block in derselben Richtung mehreren Betriebsstellen zuzuordnen, denn dann ist es quasi Random welcher Block zählt. Was geht, ist, denselben Block in zwei verschiedenen Richtungen zwei verschiedenen Betriebsstellen zuzuschlagen, also einmal links und einmal rechts. So eine Konstellation hast du ja zwischen Radolfzell und Brandbühl.

An dieser Stelle würde ich die 3 Hp's wahrscheinlich weglassen.

Reply:
Ich hab sie jetzt erstmal rausgenommen. Meine FDLs haben leider in Ihren Bereichen keine vergleichbare Konstellation. Die Variante mit den Richtungen hab ich mir auch überlegt, ich hätte es dann nach dem Motto "Erster Hp in Fahrtrichtung" belegt. Da würde die Einfahrzeit noch am Besten passen.

Reply:
Ja, das wäre ein Kompromiss. Die Ankunfts- und Abfahrtszeiten werden bei Hp's aber eh nicht gespeichert bzw. ausgewertet, weil die Blöcke meistens viel länger sind als der eigentliche Bahnsteig. Zwischen dem letzten Signal und dem eigentlichen Hp können ja durchaus mehrere Km liegen.